whois-api/src/test/java/net/ripe/db/whois/api/rest/DomainObjectServiceTestIntegration.java

```diff
@@ -230,7 +231,119 @@ public void create_domain_object_unparsable_json() {
                     "Unrecognized token 'bad': was expecting", "1", "32");
         }
     }
+    @Test
+    public void create_domain_object_fail_nserver_with_correct_prefix_ipv4() {
+
```

Comment:
can you also add a test case where nserver is not ns.ripe.net and it is allowed for /16 ?